lib/driver.js

```diff
@@ -39,6 +40,7 @@ class BaseDriver extends MobileJsonWireProtocol {
     this.shutdownUnexpectedly = false;
     this.noCommandTimer = null;
     this.shouldValidateCaps = shouldValidateCaps;
+    this.settings = new DeviceSettings();
```

Comment:
we should then pass in default settings to the driver constructor, _or_ require each driver to init its own DeviceSettings obj